orttraining/orttraining/test/python/launch_test.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import sys
+import argparse
+import subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+
+log = logging.getLogger("Build")
+
+def run_subprocess(args, cwd=None, env={}):
+    log.info("Running subprocess in '{0}'\n{1}".format(cwd or os.getcwd(), args))
+    my_env = os.environ.copy()
+    my_env.update(env)
+    completed_process = subprocess.run(args, cwd=cwd, check=True, env=my_env)
+    log.debug("Subprocess completed. Return code=" +
+              str(completed_process.returncode))
```

Comment:
given the call to subprocess.run() with check=True, won't return code always be 0 if it gets this far?

Reply:
reverted to the original method with check as a optional input

Reply:
reverted to the original subprocess